### ISSUE TITLE
Make sync:db non interactive on Travis.

### ISSUE THREAD

**Issue by ericchenshine** [acquia/blt#1841]
My system information:
* Operating system type: Ubuntu (in Travis-ci)
* Operating system version: 14.4
* BLT version: 8.9.0

We have overridden `setup` command to do `sync:db` from remote instead of `setup:drupal:install` which builds a vanilla Drupal site.

The the script is running in the Travis we saw code is hanging after running `sync:db` with the question - "Do you really want to drop all tables in the database drupal? (y/n)".

I have checked the code of `sync:db` as below

```
  public function syncDbDefault() {
    $this->invokeCommand('setup:settings');

    $local_alias = '@' . $this->getConfigValue('drush.aliases.local');
    $remote_alias = '@' . $this->getConfigValue('drush.aliases.remote');

    $task = $this->taskDrush()
      ->alias('')
      ->assume('')
      ->drush('cache-clear drush')
      ->drush('sql-drop')
      ->drush('sql-sync')
      ->arg($remote_alias)
      ->arg($local_alias)
      ->option('structure-tables-key', 'lightweight')
      ->option('create-db');

    if ($this->getConfigValue('drush.sanitize')) {
      $task->option('sanitize');
    }

    $task->drush('cache-clear drush');

    $result = $task->run();

    return $result;
  }
```

We are wondering if we could change code to 

```
    $task = $this->taskDrush()
      ->drush('cache-clear drush')
      ->drush('sql-drop')
      ->drush('sql-sync')
      ->arg($remote_alias)
      ->arg($local_alias)
      ->option('structure-tables-key', 'lightweight')
      ->option('create-db')
      ->assume(TRUE)
      ->printOutput(TRUE);
```

**Comment by grasmash** [comment]
Related to #1852.

**Comment by alex-moreno** [comment]
same issue here,  

    ->assume(TRUE)

seems to fix it

**Comment by grasmash** [comment]
Closed by #1859
